ENIGMAsystem/SHELL/Universal_System/Object_Tiers/object.h

```diff
@@ -96,21 +98,26 @@ namespace enigma
 }
 
 namespace enigma_user {
-bool object_exists(int objid);
-void object_set_depth(int objid, int val);
-void object_set_mask(int objid, int val);
-void object_set_persistent(int objid, bool val);
-void object_set_solid(int objid, bool val);
-void object_set_sprite(int objid, int val);
-void object_set_visible(int objid, bool val);
-int object_get_depth(int objid);
-int object_get_mask(int objid);
-int object_get_parent(int objid);
-bool object_get_persistent(int objid);
-bool object_get_solid(int objid);
-int object_get_sprite(int objid);
-bool object_get_visible(int objid);
-bool object_is_ancestor(int objid, int acid);
+    bool object_exists(int objid);
```

Comment:
Google style leaves these uninvented, but comments the closing brace for the namespace. I'm fine with indenting it, but prefer two spaces (like elsewhere in the project).

Reply:
Thanks for the style nit, This will be fixed in an upcoming commit.